Week3/joyeeon/Mission1/src/components/details/CastingList.tsx

```diff
@@ -0,0 +1,42 @@
+import { Credit } from "../../types/credit";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import CreditItem from "./CreditItem";
+
+
+
+const CastingList = () => {
+    const { id } = useParams();
+    const [credits, SetCredits] = useState<Credit[]>([]);
+    const token = import.meta.env.VITE_TMDB_TOKEN;
+
+    useEffect(() => {
+        const fetchCredit = async () => {
+            const { data } = await axios.get(`https://api.themoviedb.org/3/movie/${id}/credits?language=ko-KR`,
+                {
+                    headers: {
+                        Authorization: token,
```

Comment:
OAuth 2.0 표준에 따라 토큰 타입을 명시하는 "Bearer"를 접두어로 붙여주세요!

```suggestion
                        Authorization: `Bearer ${token}`,
```